tests/setup.cfg

```diff
@@ -1,2 +1,3 @@
 [tool:pytest]
 mocked-sessions=reciperadar.db.session
+mocked-sessions-propagate-connect-exceptions=false
```

Comment:
This is required to prevent exceptions from bubbling up _before_ the decorator logic in https://github.com/openculinary/backend/pull/20/files#diff-484462fced51d1a06b1d93b4a44dd535R20-R24 is able to run.